OpenXmlPowerTools/WmlToHtmlConverter.cs

```diff
@@ -528,7 +608,7 @@ private static object ProcessHyperlinkToBookmark(WordprocessingDocument wordDoc,
         {
             var style = new Dictionary<string, string>();
             var a = new XElement(Xhtml.a,
-                new XAttribute("href", "#" + (string) element.Attribute(W.anchor)),
+                new XAttribute("href", "#" + (string)element.Attribute(W.anchor)),
```

Comment:
There's a lot of formatting/style changes that should be reverted, since they only make the PR harder to review.